hledger/test/cli/report-interval.test

```diff
@@ -13,22 +13,22 @@
 # The last report interval option takes precedence.
 
 $ hledger -f- register --weekly --monthly
-2019-01                 a                                        2             2
-2019-02                 a                                        1             3
+2019-01   a                                                      2             2
```

Comment:
Any particular reason it's 3 spaces ? I think 2 is our normal minimum inter-field separator.

Reply:
I'll look at tightening that up further.

Reply:
Ah, got it. It's actually 1 space between the date and the description, a 0 width empty description, then 2 spaces between the description and the account, for 3 spaces total separation in this case.

Reply:
Ok. It looked a little weird to me but I think you're saying it's fine and good as is.